src/account/eth_account/interface.cairo

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo vX.Y.Z (account/eth_account/interface.cairo)
+
+use openzeppelin::account::interface::{ISRC6, ISRC6CamelOnly, IDeclarer, ISRC6_ID};
+use starknet::ContractAddress;
+use starknet::account::Call;
+
+type EthPublicKey = (u256, u256);
+
+#[starknet::interface]
+trait IDeployable<TState> {
+    fn __validate_deploy__(
+        self: @TState, class_hash: felt252, contract_address_salt: felt252, public_key: EthPublicKey
+    ) -> felt252;
+}
+
+#[starknet::interface]
+trait IPublicKey<TState> {
+    fn get_public_key(self: @TState) -> EthPublicKey;
+    fn set_public_key(ref self: TState, new_public_key: EthPublicKey);
+}
+
+
+#[starknet::interface]
+trait IPublicKeyCamel<TState> {
+    fn getPublicKey(self: @TState) -> EthPublicKey;
+    fn setPublicKey(ref self: TState, newPublicKey: EthPublicKey);
+}
```

Comment:
I think we don't need camel selectors because this module is not backward compatible anyway, and we didn't see any usage of the cairo 0 module in mainnet. I think we should remove the camelCase methods for the public key because of this (not for the ISRC6 interface). cc @martriay